drivers/storage/digitalocean/executor/digitalocean_executor.go

```diff
@@ -0,0 +1,120 @@
+// +build !libstorage_storage_executor libstorage_storage_executor_digitalocean
+
+package executor
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+
+	gofig "github.com/akutz/gofig/types"
+	"github.com/codedellemc/libstorage/api/registry"
+	"github.com/codedellemc/libstorage/api/types"
+	do "github.com/codedellemc/libstorage/drivers/storage/digitalocean"
+	doUtils "github.com/codedellemc/libstorage/drivers/storage/digitalocean/utils"
+)
+
+var (
+	diskPrefix = regexp.MustCompile(
+		fmt.Sprintf("^%s(.*)", do.VolumePrefix))
```

Comment:
Don't use `fmt.Sprintf` here as `do.VolumePrefix` is a constant. The code as it is malloc's two new strings just to create a regex. Since `do.VolumePrefix` is a constant do this:

```go
diskPrefix = regexp.MustCompile(`^` + do.VolumePrefix + `(.*)`)
```

Golang lacks native string interning, so I avoid creating new ones where possible when you're just going to throw them away as an initialization piece for some other object.